src/encoded/static/components/facetcharts.js

```diff
@@ -35,6 +35,7 @@ export class FacetCharts extends React.PureComponent {
     static defaultProps = {
         'href' : '/',
         'show' : function(path, search, hash){
+            if (typeof search === 'string' && search.indexOf('currentAction=impersonate-user') > -1) return false;
             if (typeof hash === 'string' && hash.indexOf('!impersonate-user') > -1) return false;
```

Comment:
Both lines say "don't show the FacetCharts if we're trying to impersonate the user." Why do we need both?

Reply:
I would say neither of these lines are necessary, since impersonate user is now done from a /user/ page and not the homepage (facet chart won't be shown there regardless)